### ISSUE TITLE
Noto Sans Mongolian Font can not able to correctly rendering some suffix in mongolian word.

### ISSUE THREAD

**Issue by siqinbilige** [googlei18n/noto-fonts#919]
### Title 
  > Noto Sans Mongolian Font can not able to correctly rendering some suffix  in mongolian word.

### Font 
  > NotoSansMongolian-Regular.ttf.

### Where the font came from, and when
  > https://noto-website.storage.googleapis.com/pkgs/NotoSansMongolian-unhinted.zip

### Font Version
  > Version 1.04 uh

### OS name and version
  > All OS

### Application name and version
  > Firefox 53.0.2 (32 bit)

### Issue
  > Noto Sans Mongolian Font can not able to correctly rendering some suffix mongolian word.

  1. prepare　following fonts.
      Mongolian Baiti which default mongolian font of Windows 10.
      Mongolian White which can download from http://www.mongolfont.com/jAlmas/cms/documents/mongolfont/font/mnglwhiteotf.ttf
  2. Access http://www.mongolfont.com/test/NotoSansMongolianFontTest/some_suffix.html
  3. Expected results like Baiti and White font
  4. this rule is not mentioned in unicode but in the mongolian grammar rule.
       the sample is better seen in firefox or ie.

### Character data
  > http://www.mongolfont.com/test/NotoSansMongolianFontTest/some_suffix.html.

### Screenshot
  > http://www.mongolfont.com/test/NotoSansMongolianFontTest/some_suffix.png
![some_suffix](https://cloud.githubusercontent.com/assets/13462439/26197156/36a54900-3bfc-11e7-901c-aadecaef64d5.png)

**Comment by siqinbilige** [comment]
For reference, see all suffix in following document.
[MongolianSuffixesSummary V1.0.pdf](https://github.com/googlei18n/noto-fonts/files/1010573/MongolianSuffixesSummary.V1.0.pdf)
This time MSC=202F
Future may be MSC=202F and 180F (maybe other) according to following request.
[17036-mongolian-suffix.pdf](https://github.com/googlei18n/noto-fonts/files/1010597/17036-mongolian-suffix.pdf)

**Comment by dougfelt** [comment]
@siqinbilige thank you for the detailed report.  Can you explain the issue with the 1826 1826 202f and 1824 1824 202f sequences?  Noto seems similar to the other fonts (within the bounds of style variation).

**Comment by siqinbilige** [comment]
@dougfelt yes, you are right. My original text was wrong one. I modified the issue.

**Comment by marekjez86** [comment]
Unicode is re-working Mongolian. I expect some results toward end of this year. We will NOT make any changes in Mongolian until we hear from the Unicode Consortium.

**Comment by kmansourMT** [comment]
@siqinbilige
Where is the re-definition for bug report 919 as stated in "I modified the issue" (dated May 18, 2017)?

What is the meaning of "MSC"?

**Comment by siqinbilige** [comment]
MSC = Mongolian Suffix Connector
For reference, see all suffix in following document.
MongolianSuffixesSummary V1.0.pdf
This time MSC=202F
Future may be MSC=202F and 180F (maybe other) according to following request.
17036-mongolian-suffix.pdf

**Comment by kmansourMT** [comment]
![screen shot 2018-05-15 at 12 24 06](https://user-images.githubusercontent.com/12738181/40078939-00f9dfe4-583b-11e8-8853-24a9fc1fdccd.png)

For patterns that start with a consonant, as in the group above, how many characters do we need to examine to determine the proper shape of the **red** character?

**Comment by kmansourMT** [comment]
![screen shot 2018-05-15 at 12 30 16](https://user-images.githubusercontent.com/12738181/40079164-c5628340-583b-11e8-898d-0dc59e63ea53.png)

Similarly, for the above patterns that start with a vowel, how many characters do we need to examine to determine the proper shape of the **red** and **green** characters?

**Comment by siqinbilige** [comment]
We need to examine all characters.
